wordpress-mobile/WordPress-Android
# Issue/reader buttons layout

### Fix
Update post layout of buttons on ***Reader*** tab as shown [here](https://scene.zeplin.io/project/5aa673f6121108413488d226/screen/5b1539a350434d3f42b62b01).  See screenshots below for illustration.

![reader_buttons](https://user-images.githubusercontent.com/3827611/41011822-4851046a-6904-11e8-8d5c-6c679e6a6d96.png)

### Test
1. Go to ***Reader*** tab.
2. Long-press ***Visit***, ***Bookmark***, ***Comment***, ***Like***, and ***Menu*** buttons on post card layout.
3. Notice behavior of buttons mimics mockups and screenshots above.
